compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -2241,6 +2241,8 @@ private void validateObjectAttachedFunction(BLangFunction funcNode) {
                         funcNode.receiver.type);
             }
 
+            dlog.error(funcNode.pos, DiagnosticCode.OBJECT_OUTSIDE_METHODS_NOT_ALLOWED);
```

Comment:
Shall we give this error at the top and return?  Other errors are not valid anymore.